std_msgs/Header header
uint32 height
uint32 width
tutorial_vision/CircleInfo[] circles

